README.rst

```diff
@@ -28,12 +28,22 @@ How can I contribute?
 What do I need to contribute?
 -----------------------------
 
-* A host with python 3.6+, gcc and the python development header files
-* tox
-* go
+* A host with `python` 3.6+ and `tox`
 * docker and/or podman+buildah
 * vagrant (optional, can be used to test FIPS mode and registered hosts)
 
+Test setup on openSUSE
+^^^^^^^^^^^^^^^^^^^^^^
+
+To setup the test environment on openSUSE Leap 15.6 and Tumbleweed, run the following commands:
+
+
+.. code-block:: shell-session
+
+    # zypper -n in podman buildah docker git-core python3 python3-pip
+    # pip3 --quiet install --upgrade pip
+    # pip3 --quiet install tox --ignore-installed six
```

Comment:
Let's suggest a user shell so that users don't wreck their whole OS
```suggestion
    $ sudo zypper -n in podman buildah docker git-core python3 python3-pip
    $ pip3 --quiet install --upgrade pip
    $ pip3 --quiet install tox --ignore-installed six
```

Reply:
Good point!